src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -103,6 +103,11 @@ class ResourceController extends Controller
     protected $eventDispatcher;
 
     /**
+     * @var string
+     */
+    protected $stateMachineGraph;
```

Comment:
Let's remove that and require to configure the graph in routing for ``updateStateAction``.